demo/demo_director.py

```diff
@@ -407,9 +407,12 @@ def register_vehicle_manifest_wrapper(
   be extracted before it is passed to the underlying director.py (in the
```

Comment:
It would probably be best to refer to variable names, yes, and yes, it refers to the .data thing. I'll clarify this docstring.